docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -42,6 +42,7 @@ keywords: ksqlDB, function, scalar
   - [MASK_RIGHT](#mask_right)
   - [REPLACE](#replace)
   - [REGEXP_EXTRACT](#regexp_extract)
+  - [REGEXP_EXTRACT_ALL](#regexp_extract_all)
```

Comment:
This TOC will go away when you resolve the merge conflicts.

Reply:
Rebased and removed this